server.js

```diff
@@ -27,11 +20,88 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
+// {
+//   "Rank": 815,
+//   "Title": "Fantastic Mr. Fox",
+//   "Genre": "Animation,Adventure,Comedy",
+//   "Description": "An urbane fox cannot resist returning to his farm raiding ways and then must help his community survive the farmers' retaliation.",
+//   "Director": "Wes Anderson",
+//   "Actors": "George Clooney, Meryl Streep, Bill Murray, Jason Schwartzman",
+//   "Year": 2009,
+//   "Runtime (Minutes)": 87,
+//   "Rating": 7.8,
+//   "Votes": 149779,
+//   "Revenue (Millions)": 21,
+//   "Metascore": 83
+// },
+
+
+const Movie = mongoose.model('Movie', {
+    Rank: {
+      type: String
+    },
+    Title: {
+      type: String
+    },
+    Genre: {
+      type: String
+    },
+    Description: {
+      type: String
+    },
+    Director: {
+      type: String
+    },
+    Actors: {
+      type: String
+    },
+    Year: {
+      type: Number
+    },
+    Runtime: {
+      type: Number
+    },
+    Rating: {
+      type: Number
+    },
+    // Revenue: {
+    //   type: Number
+    // },
+    Metascore: {
+      type: Number
+    }
+})
+
+const importMovieData = async() => {
+  await Movie.deleteMany()
+
+  movieData.forEach((movie) => {
+    new Movie(movie).save();
+  })
+}
+
+importMovieData()
```

Comment:
Careful though, this will reset and seed you database everytime the server starts.
You can control this with an env variable.